pkg/cmd/project/delete/delete.go

```diff
@@ -116,12 +116,12 @@ func deleteItemArgs(config deleteConfig) (*deleteProjectMutation, map[string]int
 	}
 }
 
-func printResults(config deleteConfig) error {
+func printResults(config deleteConfig, project queries.Project) error {
 	if !config.io.IsStdoutTTY() {
 		return nil
 	}
 
-	_, err := fmt.Fprintf(config.io.Out, "Deleted project\n")
+	_, err := fmt.Fprintf(config.io.Out, "%s\n", project.URL)
```

Comment:
Is this URL anything but a 404 at this point?

Reply:
@vilmibm that's a very good point, and probably why this function didn't originally show a URL 🤦 

I can undo this, unless you think there's some value to either
1. having the commands be consistent in their output
2. this value could be used for something else (probably a stretch)

For the first point, is there an alternative output that makes sense? My only concern is users having to remember that the delete command is different.

Reply:
Given that we're quitting if stdout is not connected to a tty I don't see value in being terse, here. I would just keep the behavior as it was before except saying something like "Deleted project; was ID 123" or something like that.

Reply:
fixed in e05262f8